client.py

```diff
@@ -421,10 +421,15 @@ def send_audio_to_server():
             file_to_send = 'recording.wav'
         with open(file_to_send, 'rb') as f:
             files = {'audio': f}
+
```

Comment:
Just picking a little and I know you are building off a starting point... But even without refactoring, let's up the documentation. Very comfortable for it to be whatever Deepseek suggests as long as you, working on the code, agree with it.

If we are doing it across the codebase, I would suggest using Aider to do it, commit after each add, etc.

Use Sphinx? Or?

Reply:
https://github.com/ClinicianFOCUS/FreeScribe/pull/3/commits/20178ae1fa7cde311afded45a50e5f8edb5f2f96

Reply:
https://github.com/ClinicianFOCUS/FreeScribe/pull/3/commits/20178ae1fa7cde311afded45a50e5f8edb5f2f96

Reply:
unless I am see an old rev, docs still need work - but not going to ask for revisions just on this.  Also formatting; I see extra white space, etc.